app/javascript/dashboard/api/ApiClient.js

```diff
@@ -41,6 +41,10 @@ class ApiClient {
     return axios.post(this.url, data);
   }
 
+  createAccount(data) {
```

Comment:
we can move this method to `api/account.js`

Reply:
done